osidb/serializer.py

```diff
@@ -623,6 +624,32 @@ def to_representation(self, value):
         return list(trackers)
 
 
+class FlawAcknowledgmentSerializer(
```

Comment:
`FlawAcknowledgmentSerializer` should inherit also from `IncludeMetaAttrMixin`.

Reply:
Fixed in commit 3c2d2dd147380fc8924fac4b9d539f85f86f36d4